apps/arena/lib/arena/game_updater.ex

```diff
@@ -662,6 +692,17 @@ defmodule Arena.GameUpdater do
     PubSub.broadcast(Arena.PubSub, state.game_id, {:game_update, encoded_state})
   end
 
+  defp broadcast_ping(state) do
+    now = DateTime.utc_now() |> DateTime.to_unix(:millisecond)
+
+    encoded_state =
+      GameEvent.encode(%GameEvent{
+        event: {:ping, %Ping{timestamp_now: now}}
+      })
+
+    PubSub.broadcast(Arena.PubSub, state.game_id, {:ping, encoded_state})
```

Comment:
Broadcasting here may be wrong in the ping context since every client will have a different ping

Reply:
You're totally right. Nice catch.
Fixed!